hyco-ws/index.d.ts

```diff
@@ -0,0 +1,27 @@
+// Type definitions for hyco-ws 1.0
+// Project: https://github.com/Azure/azure-relay-node
+// Definitions by: Manuel Rodrigo Cabello <https://github.com/mrcabellom>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+import * as WebSocket from 'ws';
+
+export class HybridConnectionWebSocketServer extends NodeJS.EventEmitter {
+    constructor(options: any);
```

Comment:
Thanks! I'm waiting for Microsoft. We need more doc about this new library for hybrid connections.